<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>331</width>
      <height>511</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFileBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Product name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Income date:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printListBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Print List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printWaitListBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>600</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Wait List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWaitBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To Wait List</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchByNameLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchByNameBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search By Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchByDateLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchByDateBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search By Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>display goods stored greater than the entered value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchByPriceBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search By Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchByPriceLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>displays products whose cost is lower than the entered value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete From LIst</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popWaitBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete From Wait List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase From LIst</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inisertBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert New Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
